build-packages/magento-scripts/lib/commands/start.js

```diff
@@ -99,9 +103,31 @@ module.exports = (yargs) => {
             logger.logN(`Magento Admin panel credentials: ${logger.style.misc(magentoConfiguration.user)} - ${logger.style.misc(magentoConfiguration.password)}`);
             logger.note(`MySQL credentials, containers status and project information available in ${logger.style.code('npm run status')} command.`);
             logger.log('');
+
+            if (!analytics) {
+                process.exit(0);
+            }
+
+            try {
+                if (!process.isFirstStart) {
+                    await googleAnalytics.trackTiming('CMA start time', Date.now() / 1000 - timeStamp);
+                    process.exit(0);
+                }
+
+                // Get ram amount in MB
+                const ramAmount = Math.round(os.totalmem() / 1024 / 1024);
+                const cpuModel = os.cpus()[0].model;
+
+                await googleAnalytics.trackTiming('CMA first start time', Date.now() / 1000 - timeStamp);
+                await googleAnalytics.trackEvent('Params', `Platform: ${os.platform}, CPU model: ${cpuModel}, RAM amount: ${ramAmount} MB`, 0, 'OS');
```

Comment:
Also, those asynchronous functions, if they throw an error, should we really print those errors?